test/feature/step_definitions/eval_definitions.coffee

```diff
@@ -5,23 +6,29 @@ module.exports = ->
 
   @Then /^I have a routing helper "([^"]+)" that returns "([^"]+)"$/, (helper, value, done) ->
     @makeRequest 'get', "/eval/return String(#{helper})", (err, _, responseBody) ->
-      return done.fail err if err
+      return done err if err
       expect(responseBody).to.equal value
       done()
 
 
   @Then /^I have a routing helper "([^"]+)" that returns "([^"]+)" and the HTTP verb "([^"]+)"$/, (helper, value, method, done) ->
-    @makeRequest 'get', "/eval/return String(#{helper})", (err, _, responseBody) =>
-      return done.fail err if err
-      expect(responseBody).to.equal value
-      @makeRequest 'get', "/eval/return String(#{helper}.method)", (err, _, responseBody) ->
-        return done.fail err if err
-        expect(responseBody).to.equal method
-        done()
+    async.parallel [
+      (next) =>
+        @makeRequest 'get', "/eval/return String(#{helper})", (err, _, responseBody) ->
+          return next err if err
```

Comment:
If an error is a failure here that should cause the test to fail, you can use https://github.com/Originate/nitroglycerin to clean this up even further.

```coffeescript
@makeRequest 'get', "/eval/return String(#{helper})", N (err, _, responseBody) ->
  expect(responseBody).to.equal value
  ...